CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ set(libraries
     toolbox_common
     SlamToolboxPlugin
     ceres_solver_plugin
+    multirobot_slam_toolbox
```

Comment:
For this + the Class + file names: I think we need to make this more specific since we'll have both nodes in the project. 

Suggestion: Decentralized Multi-robot?